test/integ/pandas/test_pandas_tools.py

```diff
@@ -43,6 +43,117 @@
 )
 
 
+@pytest.mark.parametrize("quote_identifiers", [True, False])
+@pytest.mark.parametrize("overwrite", [True, False])
```

Comment:
I think `test_write_pandas` is already testing the `overwrite=False` case. Could we extend that test to verify that specific behavior? If we get rid of this one parameter then we'd go from 8 -> 4 tests in each of which you call `write_pandas` 4 times.